QModPluginEmulator/QModManager.QModPluginGenerator.csproj

```diff
@@ -43,23 +63,22 @@
       <HintPath>..\Dependencies\BepInEx\BepInEx\core\Mono.Cecil.dll</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="Newtonsoft.Json">
-      <HintPath>..\Dependencies\Newtonsoft.Json.dll</HintPath>
+    <Reference Include="Newtonsoft.Json, Version=12.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
+      <HintPath>..\packages\Newtonsoft.Json.12.0.1\lib\net45\Newtonsoft.Json.dll</HintPath>
```

Comment:
Any reason why we are using NuGet to fetch `Newtonsoft.Json` and `Newtonsoft.Json.Bson` on this config?

Reply:
Not that I can see. .. Removing them